src/locales/en/main.json

```
@@ -593,6 +593,12 @@
     "Bypass": "Bypass",
     "Copy (Clipspace)": "Copy (Clipspace)",
     "Add Node": "Add Node",
+    "Add Reroute": "Add Reroute",
+    "Most Relevant": "Most Relevant",
+    "Comfy Nodes": "Comfy Nodes",
+    "Extensions": "Extensions",
```

Biome (2.4.16)

[error] The key Extensions was already declared.

(lint/suspicious/noDuplicateObjectKeys)